### ISSUE TITLE
Not a bug but a discrepency between the log and config file for kd-resnet18_from_resnet34 

### ISSUE THREAD

**Issue by Coderx7** [yoshitomo-matsubara/torchdistill#279]
**Describe the bug**
Looking at the config file and its corresponding log file for resnet18, [kd-resnet18_from_resnet34.log](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/official/ilsvrc2012/yoshitomo-matsubara/rrpr2020/kd-resnet18_from_resnet34.log), it seems the provided config file is different from the one being used to create the log file. 
If you look closely you can see in the log file that, the learning rate is 0.3 while in the config file its set as 0.1.

**Comment by yoshitomo-matsubara** [comment]
Hi @Coderx7 

See https://github.com/yoshitomo-matsubara/torchdistill/issues/237
It is not a bug. The learning rate was adjusted (multiplied by number of GPUs) for distributed training mode with 3 GPUs using [`-adjust_lr`](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/official/ilsvrc2012/yoshitomo-matsubara/rrpr2020/kd-resnet18_from_resnet34.log#L2)
Please check closed issues and discussions next time, before you create a new issue/discussion
